File: java/lld/marketplace/food-delivery/Courier.java
// Same shape as Cab Booking's Driver — the matching machinery is reused,
// not reinvented, for a completely different kind of trip.
public class Courier {
    public enum Status { OFFLINE, AVAILABLE, EN_ROUTE_TO_PICKUP, ON_DELIVERY }

    public final String id;
    public double lat, lng;
    public Status status = Status.OFFLINE;

    public Courier(String id, double lat, double lng) {
        this.id = id; this.lat = lat; this.lng = lng;
    }
}

File: python/lld/marketplace/food-delivery/courier.py
from enum import Enum


class CourierAvailability(Enum):
    """Same shape as Cab Booking's Driver — the matching machinery is
    reused, not reinvented, for a completely different kind of trip."""
    OFFLINE = "OFFLINE"
    AVAILABLE = "AVAILABLE"
    EN_ROUTE_TO_PICKUP = "EN_ROUTE_TO_PICKUP"
    ON_DELIVERY = "ON_DELIVERY"


class Courier:
    def __init__(self, id: str, lat: float, lng: float):
        self.id = id
        self.lat = lat
        self.lng = lng
        self.status = CourierAvailability.OFFLINE